### PR TITLE
fix: allow getBankNameFromCardNumber for at least 6 num

### PR DESCRIPTION
**Pull request by danialnoaein**
To find bank name from card number just need the first six numbers

Resolve issue #257

### CODE REVIEW

File: src/modules/getBankNameFromCardNumber/index.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export interface IBank {
 function getBankNameFromCardNumber(digits?: number | string): string | null | undefined {
 	if (!digits) return;
 
-	if (digits && digits.toString().length === 16) {
+	if (digits && digits.toString().length >= 6 && digits.toString().length <= 16) {
```
Comment (ali-master) [inline src/modules/getBankNameFromCardNumber/index.ts]: `digits.toString().length`called twice.

**Review by ali-master — CHANGES_REQUESTED** [review]

File: src/modules/getBankNameFromCardNumber/index.ts (hunk 1 above)
Comment (danialnoaein) [reply src/modules/getBankNameFromCardNumber/index.ts]: Do you think it's okay to define a const for the digits length?

Comment (ali-master) [reply src/modules/getBankNameFromCardNumber/index.ts]: Exactly

**Comment by ali-master** [comment]
Welcome to the `PersianTools` community. Happy to see you here.

**Pull request merged**